information_for_beamlines_ds/information_for_beamlines.py

```diff
@@ -143,25 +141,25 @@ def state_from_data(self, data):
         self.set_state(DevState.ON)
         self.set_status('Everything is OK.')
 
-        # ------------------
-        # Attributes methods
-        # ------------------
+    # ------------------
+    # Attributes methods
+    # ------------------
 
     def read_GeneralInfo(self):
         return self.general_info
 
     def write_GeneralInfo(self, value):
         self.general_info = value
-        # self.push_change_event('GeneralInfo')
-        # self.push_data_ready_event('GeneralInfo')
+        self.push_change_event('GeneralInfo', value)
+        self.push_data_ready_event('GeneralInfo', 0)
 
     def read_FillingPattern(self):
         return self.filling_pattern
 
     def write_FillingPattern(self, value):
         self.filling_pattern = value
-        # self.push_change_event('FillingPattern')
-        # self.push_data_ready_event('FillingPattern')
+        self.push_change_event('FillingPattern', value)
+        self.push_data_ready_event('FillingPattern', 0)
```

Comment:
Please check subscription to these attributes (you'll only see events appearing after writing something to these attributes).